prism/src/main/java/org/apache/falcon/service/FeedSLAMonitoringService.java

```diff
@@ -425,10 +423,10 @@ private void serializeState() throws FalconException{
     }
 
     @SuppressWarnings("unchecked")
+    //Need to check this method
```

Comment:
Checked :) Deserialization is not needed anymore, so we can remove serialize, deserialize, (their internal methods) etc. We will also need to remove usage of the fields which were populated by these.